sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.0.0b3 (Unreleased)
+
+### Features Added
+
+* Redesigned streaming and event handlers for agents 
+* Add `parallel_tool_calls` parameter to allow parallel tool execution.
+* Add support for Structured Outputs.
+* Add option to include file contents, when index search is used.
+
```

Comment:
Make sure "for Agents" is mentioned in every line

Reply:
Fixed.